vignettes/load-scraped-data.Rmd

```diff
@@ -0,0 +1,99 @@
+author: "Jason Zivkovic"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Extracting data from FBref}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  message=FALSE,
+  warning=FALSE
+)
+```
+
+
+## Overview
+
+This package is designed to allow users to extract various world football results and player statistics from the following popular football (soccer) data sites:
+
+* [FBref](https://fbref.com/en/)
+* [Transfermarkt](https://www.transfermarkt.com/)
+* [Understat](https://understat.com/)
+* [Fotmob](https://www.fotmob.com/)
+
+
+## Installation
+
+You can install the CRAN version of [**```worldfootballR```** ](https://CRAN.R-project.org/package=worldfootballR) with:
+
+```{r cran-installation, eval=FALSE}
+install.packages("worldfootballR")
+```
+
+You can install the released version of [**```worldfootballR```**](https://github.com/JaseZiv/worldfootballR/) from [GitHub](https://github.com/JaseZiv/worldfootballR) with:
+
+```{r gh-installation, eval=FALSE}
+# install.packages("devtools")
+devtools::install_github("JaseZiv/worldfootballR")
+```
+
+
+```{r load_libs, warning=FALSE, message=FALSE}
+library(worldfootballR)
+```
+
+***
+
+## Usage
+
+Package vignettes have been built to help you get started with the package.
+
+* For functions to extract data from Transfermarkt, see [here](https://jaseziv.github.io/worldfootballR/articles/extract-transfermarkt-data.html)
+* For functions to extract data from Understat, see [here](https://jaseziv.github.io/worldfootballR/articles/extract-understat-data.html)
+* For functions to extract data from fotmob, see [here](https://jaseziv.github.io/worldfootballR/articles/extract-fotmob-data.html)
+* For functions to extract data for international matches from FBref, see [here](https://jaseziv.github.io/worldfootballR/articles/fbref-data-internationals.html)
+* For functions to load pre-scraped data, see [here](https://jaseziv.github.io/worldfootballR/articles/load-scraped-data.html)
+
+This vignette will cover the functions to load scraped data from the [`worldfootballR_data`](https://github.com/JaseZiv/worldfootballR_data) data repository.
+
+**NOTE:**
+
+As of version `0.5.2`, all FBref functions now come with a user-defined pause between page loads to address their new rate limiting. See [this](https://www.sports-reference.com/bot-traffic.html) document for more information. 
+
+
+***
+
+## Load FBref match results
+
+To load pre-scraped match results for all years the data is available the `load_match_results()` function can be used. This data is scheduled to be updated most days and a print statement will inform the user of when the data was last updated. All domestic leagues are included in the data repository. 
+
+This is the load function equivalent of `get_match_results()`.
+
+```{r load_match_results, eval=FALSE}
```

Comment:
why `eval=FALSE`?

Reply:
Because the vignette loads the library at the start, then fails to recognise these new functions as they're not yet deployed to the library. So once the version gets updated, then I go in and rebuild that one article and set `eval=TRUE`...

Unless I've got that completely wrong?

Reply:
if we weren't ignoring vignettes with `.Rbuildignore`, I believe the functions would be available to the vignettes anyways. i think the package gets built before the vignettes